src/components/ShortcutsModal.css

```diff
@@ -41,7 +41,7 @@
 }
 
 .shortcuts-modal {
-  height: fit-content;
+  display: table;
```

Comment:
this is pretty esoteric, how did you come up w/ it?

could you link to a stack overflow or other known issue w/ firefox and fit-content

e.g.
```
// NOTE firefox does not support `height: fit-content`
// stackoverrflow.com/sldkjfslfjsljsldflsdjslkdjfls
display: table;
```

Reply:
Yeah moz-fit-content didn't seem to do the same thing as fit-content
https://stackoverflow.com/questions/7839164/is-there-a-css-cross-browser-value-for-width-moz-fit-content